README.md

```diff
@@ -51,9 +63,14 @@ launchInto(Workflow(ExampleViewController.self))
 And just like that you're started!  To see something more practical and in-depth, check out the example app in the repo.  For a more in-depth starting guide, checkout out our [Getting Started](https://github.com/wwt/SwiftCurrent/wiki/getting-started) documentation.
```

Comment:
It is a practical example, as we believe this would best suit consumer's needs. The Getting Started app includes detailed explanations for a trivial. 
 
 For a quick start, we've included an example app that includes detailed comments and explanations.

Reply:
Thanks for spotting this section.  I completely forgot to update it when I added the new call-out at the top.